docs/index.md

```diff
@@ -28,17 +14,16 @@ title: Home
       <div class="govuk-grid-column-one-third govuk-!-margin-bottom-8">
         <h2 class="govuk-heading-l">Components</h2>
         <p>Save time with reusable, accessible components for forms, navigation, panels, tables and more.</p>
-        <p><a class="app-link govuk-!-font-weight-bold" href="./components">Browse components</a></p>
+        <p><a class="app-link govuk-!-font-weight-bold" href="/components">Browse components</a></p>
```

Comment:
This needs to remain as `./components` otherwise it won't work when hosted on GitHub Pages (because they have a prefix of `/moj-frontend`)
```suggestion
        <p><a class="app-link govuk-!-font-weight-bold" href="./components">Browse components</a></p>
```

Reply:
Done